modules/ROOT/pages/release-notes/release-notes5013.adoc

```diff
@@ -24,14 +24,14 @@ TinyMCE now supports all valid HTML color formats for the `color_map` setting. F
 
```

Comment:
The list above isn't rendering correctly as it needs a blank line between the para and list items.

Reply:
done